example/lib/main.dart

```diff
@@ -108,13 +108,13 @@ class _Loading extends StatelessWidget {
 
 class _Error extends StatelessWidget {
   const _Error({
-    Key key,
+    Key? key,
     this.error,
     this.retry,
   }) : super(key: key);
 
-  final Object error;
-  final VoidCallback retry;
+  final Object? error;
+  final VoidCallback? retry;
```

Comment:
can we make these required instead of nullable?

Reply:
yes we should make this 
because typedef has this two
```typedef ErrorBuilder = Widget Function(
  BuildContext context,
  VoidCallback retry,
  Object error,
);```

Reply:
should we make `required` here also?
<img width="390" alt="image" src="https://user-images.githubusercontent.com/56479665/107905240-2609d480-6f74-11eb-989e-a30c4b9c7f6f.png">

Reply:
@felangel Please give a thought on it also.

Reply:
yup I think we should 👍